server/src/assets/game/tableGenerator/generator.py

```diff
@@ -1,38 +1,40 @@
-import json
-import csv
+import json, csv, os
 
 def numberToWord(number):
     return ["Zero", "One", "Two", "Three", "Four", "Five", "Six"][int(number)]
 
 def characterToColor(character):
-	return {"b": "Blue", "g": "Green", "r": "Red", "s": "Black"}[character]
-
-# Load Match to Signal into File
-with open("flags-swtbahn-full.csv", "r") as f:
-    reader = csv.reader(f)
-    jsonString = {}
-    for row in reader:
-        signal = row[0]
-        colorDefinition = row[1]
-        number = colorDefinition[-1]
-
-        endString = "flagTheme" + characterToColor(colorDefinition[0])
-
-        if colorDefinition[1] == "w":
-            endString += " flagOutline "
-        else:
-            endString += " flagFilled "
-
-        if int(number) == 6:
-            number = 0
-
-        endString += "flag" + numberToWord(number)
-        print(endString)
-        signalName = "signal{}".format(str(signal))
-        jsonString[signalName] = endString
-
-    with open("../flags-swtbahn-full.json", "w") as file:
-        file.write("const signalFlagMapSwtbahnFull = ")
-        file.write(json.dumps(jsonString, indent=2))
-        file.write(";")
-
+	return {"b": "Blue", "g": "Green", "r": "Red", "s": "Black"}[character.lower()]
+
+groupingFileDirectory = "./flagMappings"
+    
+mappingFolderContent = os.scandir(groupingFileDirectory)
+for entry in mappingFolderContent:
+    if entry.is_file() and entry.name[-4:] == ".csv":
+        with open("{}/{}".format(groupingFileDirectory, entry.name), "r") as f:
+            reader = csv.reader(f)
+            jsonString = {}
+            for row in reader:
+                signal = row[0]
+                colorDefinition = row[1]
+                number = colorDefinition[-1]
+
+                endString = "flagTheme" + characterToColor(colorDefinition[0])
+
+                if colorDefinition[1] == "w":
+                    endString += " flagOutline "
+                else:
+                    endString += " flagFilled "
+
+                if int(number) == 6:
+                    number = 0
```

Comment:
Why this? `numberToWord(6)` would return "Six".

`number %= 7`

Reply:
currently we dont have six as symbol